rfc/0056-sdk3-query.md

```diff
@@ -103,6 +103,14 @@ QueryResult Query(string statement, [QueryOptions options]);
     * Specifies that the query engine should preserve expiration values set on any documents modified by this query.
     * Sent within the JSON Payload as `preserve_expiry` as a JSON boolean
     * Should be added at API stability level of uncommitted.
+  * useReplica(bool) = undefined
+    * Specifies that the query engine should use replica nodes for kv fetches if the active node is down.
+    * Sent within the JSON payload as `use_replica` as the JSON string `"on"` or `"off"`, only if set by the user.
+    * If the SDK does not have the ability to differentiate between not set and `false` then a different type should be used instead, suggested:
+      * `enum QueryUseReplicaLevel { ON, OFF }` 
+    * When `useReplica` is set by the user the SDK should check the `clusterCapabilities` to ensure that there is a `niql` field containing a `readFromReplica` entry.
```

Comment:
Shouldn't this be 'useReplicas' rather than 'useReplica'?  Since there are potentially multiple replicas that might be used?

Reply:
Query call it use_replica as the parameter so I think useReplica is OK.